webapp/channels/src/components/user_profile/user_profile.tsx

```diff
@@ -92,7 +90,7 @@ export default class UserProfile extends PureComponent<UserProfileProps> {
 
         const ariaName: string = typeof name === 'string' ? name.toLowerCase() : '';
 
-        let userColor = '#000000';
+        let userColor = theme?.centerChannelColor;
```

Comment:
Note this is the real change, the remaining stuff in this PR is code clean up, the removal unused components props